ensemble/ctf/editor.py

```diff
@@ -220,14 +227,14 @@ def _draw_histogram(self, gc):
         values = values.astype(float)
         zeros = (values == 0)
         min_nonzero = values[~zeros].min()
-        values[zeros] = min_nonzero / 2.0
+        values[zeros] = min_nonzero / 2
         log_values = np.log(values)
         log_values -= log_values.min()
         log_values /= log_values.max()
 
         h_values = log_values * h
         bin_edges = bin_edges - bin_edges.min()
-        bin_edges *= w / bin_edges.max()
+        bin_edges *= w // bin_edges.max()
```

Comment:
This is the only change i'm not sure about. I'm not sure if the `width` attr of an enable `Container` is always an int or if it can be a float.

Reply:
I've reverted this change. The division isn't a floor division anymore.